README.md

```diff
@@ -88,6 +88,8 @@ Possible options that can be passed to FortySecondsCV are:
   columns.
 * `leftrightmargin=<length>` sets the left and right page margin for both
   columns as well as how much space will be between both columns.
+* `nameabovepicture` will put the name and job title below the profile picture
+  instead of above it.
```

Comment:
Isn't it the other way around? Setting the option will place name and job *above* instead of below the profile picture.

Reply:
Again, you're completely right. I revised the doc change since I changed a bit the option.